test/Verify.t.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+import {Test, console2} from "forge-std/Test.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {Verify} from "../src/Verify.sol";
+import {P256Verifier} from "../src/P256Verifier.sol";
+
+contract VerifyTest is Test {
+    uint256[2] public pubKey;
+
+    function setUp() public {
```

Comment:
recommend CREATE2-deploying P256Verifier in setUp()

two advantages:
- test reliability: tests run offline, dont rely on `fork-url` / the public base-goerli endpoint flakes sometimes
- sanity checks that the hardcoded `VERIFIER` address and create2-deployed `P256Verifier` address match

Reply:
seems running CREATE2 in tests generates a different address. Worth investigating in more detail but I suspect it's a Foundry bug as we discussed. Maybe we should just pull this out into its own issue for future?

re public base-goerli endpoint flakes: I've been using a different RPC (locally and in this PR) which hasn't flaked for me since I switched

Reply:
Moved to #22. Merging as-is.